.github/workflows/merge-publish.yml

```diff
@@ -35,11 +35,9 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-        
-      - name: branch name tag
-        run: | 
-          BRANCH="${GITHUB_REF#refs/heads/}"
-          echo "BRANCH=${BRANCH}" >> $GITHUB_ENV
 
       - name: push docker image
-        run: ./gradlew dockerPushImages
\ No newline at end of file
+        run: ./gradlew dockerPushImages
```

Comment:
Do we still `docker login` after adding env here? If, not, we can remove that in follow-up PR.